app/celery/letters_pdf_tasks.py

```diff
@@ -170,6 +179,58 @@ def collate_letter_pdfs_to_be_sent():
     current_app.logger.info("finished collate-letter-pdfs-to-be-sent")
 
 
+def _get_letters_and_sheets_volumes_and_send_to_dvla(print_run_deadline):
+    letters_volumes = dao_get_letters_and_sheets_volume_by_postage(print_run_deadline)
+    send_letters_volume_email_to_dvla(letters_volumes)
+
+
+def send_letters_volume_email_to_dvla(letters_volumes):
+    personalisation = {
+        'total_volume': 0,
+        'first_class_volume': 0,
+        'second_class_volume': 0,
+        'international_volume': 0,
+        'total_sheets': 0,
+        'first_class_sheets': 0,
+        "second_class_sheets": 0,
+        'international_sheets': 0
+    }
+    for item in letters_volumes:
+        personalisation['total_volume'] += item.letters_count
+        personalisation['total_sheets'] += item.sheets_count
+        if f"{item.postage}_class_volume" in personalisation:
+            personalisation[f"{item.postage}_class_volume"] = item.letters_count
+            personalisation[f"{item.postage}_class_sheets"] = item.sheets_count
+        else:
```

Comment:
Did DVLA ask for the rest-of-world and europe volumes to be aggregated as international?

Reply:
they have slight preference for aggregated, but also they said whatever is easier for us.

Reply:
okay